### PR TITLE
edit knowledge graph to 80%

### PR DESCRIPTION
**Pull request by allaway**
### Purpose

addresses #87 

### Directions for reviewers

Review to see if this section still makes sense. 

#### Which areas should receive a particularly close look?

Did I cut too aggressively? This was a short section already and I had trouble deleting content but still having it make sense. 

#### Is there anything that you want to discuss further?

see above

#### Is the pull request ready for review?

Yes

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [x] All terms that need to be added to glossary have been added to #50 
- [x] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/03.prior-knowledge.knowledge-graphs.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
-### Knowledge graphs for rare disease
+### Knowledge graphs 
+An intrinsic constraint in studying rare diseases is the lack of large, normalized datasets, which limits our ability to study key attributes of rare diseases. 
+A potentially powerful strategy for evaluating genotype-phenotype relationships or repurposing drugs when large datasets are scarce is to use knowledge graphs (KGs).
```
Comment (jaclyn-taroni) [inline content/03.prior-knowledge.knowledge-graphs.md]: I am of the opinion that using KGs as an acronym is going to be more trouble than it's worth – the sacrifice of readability will not be worth the words saved. (This is why I don't use LVs for latent variables in the transfer learning section.)

File: content/03.prior-knowledge.knowledge-graphs.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-In the academic rare disease space, there a few pioneering examples of machine learning-based mining of knowledge graphs to repurpose drugs[@doi:10.1101/727925] and classify rare diseases[@doi:10.1186/s12911-019-0938-1].
-These studies make it clear that there are some challenges in using machine learning using graph databases in rare disease.
-For example, these papers rely on a gold standard dataset to validate the performance of the models; often, there are not robust gold standard datasets available for individual rare diseases.
-These methods also evaluate a broad swath of rare diseases in a relatively unguided manner, rather than interrogating a pre-defined disease of interest.
-Consequently, it is not yet clear how effective these approaches, and knowledge graphs in general, are in studying a specific disease of interest; more work needs to be done to identify methods that can provide actionable insights for a specific rare disease application. 
-Beyond the aforementioned studies, there are very few examples of studies in the public domain that leverage knowledge graphs to characterize rare disease.
```
Comment (jaclyn-taroni) [inline content/03.prior-knowledge.knowledge-graphs.md]: I do feel like this section in particular may have been cut too aggressively.

**Review by jaclyn-taroni — COMMENTED** [review]
Agree that you may have cut a bit too aggressively here! In particular, I feel like the challenges uncovered in the academic rare disease space lacks some context with these cuts.

File: content/03.prior-knowledge.knowledge-graphs.md (hunk 1 above)
Comment (allaway) [reply content/03.prior-knowledge.knowledge-graphs.md]: thanks, i was borderline on the idea, so definitely happy to change this back!

File: content/03.prior-knowledge.knowledge-graphs.md (hunk 2 above)
Comment (allaway) [reply content/03.prior-knowledge.knowledge-graphs.md]: Added back a bit of content here. I'm not sure what is going on with the diff when I first filed the PR, but I didn't change all that much about these lines...

**Review by jaclyn-taroni — APPROVED** [review]
:shipit:

**Pull request merged**